ipaserver/install/installutils.py

```diff
@@ -436,16 +436,27 @@ def format_directive(directive, value, separator, quotes, quote_char):
     fd.close()
     os.chown(filename, st.st_uid, st.st_gid) # reset perms
 
-def get_directive(filename, directive, separator=' '):
+
+def get_directive(filename, directive, separator=' ', quote_char='\"'):
     """
     A rather inefficient way to get a configuration directive.
+
+    :param filename: input filename
+    :param directive: directive name
+    :param separator: separator between directive and value
+    :param quote_char: the characters that are used in this particular config
+        file to quote values. This character will be stripped and unescaped
+        from the raw value.
+
+    :returns: The (unquoted) value if the directive was found, None otherwise
     """
     fd = open(filename, "r")
     for line in fd:
         if line.lstrip().startswith(directive):
             line = line.strip()
             result = line.split(separator, 1)[1]
-            result = result.strip('"')
+            result = result.strip(quote_char)
```

Comment:
Thinking about it a bit more, I would argue that first use of `strip()` on line https://github.com/freeipa/freeipa/pull/347#discussion-diff-92979573R456 is incorrect anyway. The code first does `strip()` and then `split(separator, 1)[1]`.

What if result of `split(separator, 1)[1]` contains whitespace?